devbuild.sh

```diff
@@ -29,12 +27,20 @@ OPTIONS
       (e.g. 32BIT | INLINE_POST | UFS_GOCART | MOM6 | CICE6 | WW3 | CMEPS)
   --continue
       continue with existing build
-  --clean
+  --remove
       removes existing build; overrides --continue
+  --clean
+      does a "make clean"
+  --build
+      does a "make"
+  --install
+      does a "make install"
   --build-dir=BUILD_DIR
       build directory
   --install-dir=INSTALL_DIR
       installation prefix
+  --bin-dir=BIN_DIR
+      installation binary directory name ("bin" or "exec")
```

Comment:
Are these the only two options? Could this be set arbitrarily?

Reply:
Unfortunately no. UPP and UFS_UTILS other projects do not use `CMAKE_INSTALL_BINDIR` as the destination target, instead they set to either "bin" or "exec" where the latter is used if an "EMC_EXEC_DIR" variable is turned on. But once these are fixed, we should be able to install to any bind directory name.